### PR TITLE
Add jgitver to project

### PR DESCRIPTION
**Pull request by hugomiguelabreu**

### CODE REVIEW

File: .m2/settings.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!--
+  ~ Copyright (c) 2018 Feedzai
```
Comment (TiagoMSSantos) [inline .m2/settings.xml:2]: 2018?

File: .m2/settings.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+
+<settings>
+    <servers>
+        <server>
+            <id>sonatype-nexus</id>
+            <username>${env.SONATYPE_USER}</username>
```
Comment (TiagoMSSantos) [inline .m2/settings.xml:21]: Where is the `SONATYPE_USER` defined?

File: .mvn/extensions.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
+    <extension>
+        <groupId>fr.brouillard.oss</groupId>
+        <artifactId>jgitver-maven-plugin</artifactId>
+        <version>1.5.1</version>
```
Comment (TiagoMSSantos) [inline .mvn/extensions.xml:22]: Why not 1.7.1? https://mvnrepository.com/artifact/fr.brouillard.oss/jgitver-maven-plugin

File: pom.xml [hunk 4]
```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.feedzai</groupId>
     <artifactId>pdb</artifactId>
-    <version>2.8.7-SNAPSHOT</version>
+    <version>0.0.0-SNAPSHOT</version>
```
Comment (TiagoMSSantos) [inline pom.xml:6]: I'm guessing this jgitver will automatically discover the next SNAPSHOT version when we do a materialize git commit, right?

File: .m2/settings.xml (hunk 2 above)
Comment (hugomiguelabreu) [reply .m2/settings.xml:21]: It should be on the machine environment

File: pom.xml (hunk 4 above)
Comment (hugomiguelabreu) [reply pom.xml:6]: Not the git commit. It's based on the tags.

**Review by TiagoMSSantos — APPROVED** [review]
lgtm

**Pull request merged**